### ISSUE TITLE
Consider updating tooltips used in EDD

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#8074]
### Explain your enhancement (please be detailed)
This came up while doing some research for updating a setting for EDD Stripe, but I think it would be good if we reviewed our use of tooltips (primarily in the admin; I don't know offhand if we use them in the front end).

### Justification or use case
This article on [Tooltips and Toggletips](https://inclusive-components.design/tooltips-toggletips/) is very thorough in discussing how to create accessible tooltips. The way we implement tooltips is not accessible, so when they are needed, we could improve that experience.

Additionally, we could review our overall language and see if we even _need_ tooltips in all the places we are using them, or if we could update the information displayed. A quote from the article:

> As we already established, tooltips are for clarification; they are for providing missing information. But why should that information be missing in the first place?

Our conclusion for the specific Stripe scenario was that the information didn't need to be missing--it's possible that is true in other cases as well.
